<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QWurthEasyScan</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="1">
     <widget class="QPushButton" name="pushButtonList">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButtonQueryDevice">
      <property name="text">
       <string>Query device</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButtonReadBarcodes">
      <property name="text">
       <string>Read barcodes from device</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" rowspan="3">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButtonClearBarcodes">
      <property name="text">
       <string>Clear barcodes from device</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="pushButtonSaveList">
      <property name="text">
       <string>Save to file</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="6" colspan="2">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButtonPowerDown">
      <property name="text">
       <string>Power down the device</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>447</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuPort">
     <property name="title">
      <string>Port</string>
     </property>
     <addaction name="actionPlaceholder"/>
    </widget>
    <addaction name="menuPort"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_QWurthScanner"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLogging">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Logging</string>
   </property>
  </action>
  <action name="actionShow_log">
   <property name="text">
    <string>Show log</string>
   </property>
  </action>
  <action name="actionPlaceholder">
   <property name="text">
    <string>Placeholder</string>
   </property>
  </action>
  <action name="actionAbout_QWurthScanner">
   <property name="text">
    <string>About QWurthScanner</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
